### PR TITLE
fix: typo

### PR DESCRIPTION
**Pull request by mordechairoth**
removed an extra (misspelled) word.

## Description of Problem
There was an extra and misspelled word "a11y" in the docs.
## Proposed Solution
Remove those extra characters.

### CODE REVIEW

**Comment by skirtles-code** [comment]
This is intentional. `a11y` is an abbreviation of `accessibility`. I wonder whether it would be better to write out the word in full though, especially given the lack of surrounding context.

**Comment by mordechairoth** [comment]
@skirtles-code I think it might be helpful to write out the word `accessibility` in full, as you mentioned. It might make this doc more 'accessibly' to people like me and other 😉.

File: src/guide/extras/rendering-mechanism.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Vue templates are compiled into virtual DOM render functions. Vue also provides
 
 So why does Vue recommend templates by default? There are a number of reasons:
 
-1. Templates are closer to actual HTML. This makes it easier to reuse existing HTML snippets, apply a11y best practices, style with CSS, and for designers to understand and modify.
+1. Templates are closer to actual HTML. This makes it easier to reuse existing HTML snippets, apply best practices, style with CSS, and for designers to understand and modify.
```
Comment (NataliaTepluhina) [inline src/guide/extras/rendering-mechanism.md:57]:
```suggestion
1. Templates are closer to actual HTML. This makes it easier to reuse existing HTML snippets, apply accessibility best practices, style with CSS, and for designers to understand and modify.
```

**Comment by NataliaTepluhina** [comment]
@mordechairoth @skirtles-code I believe you both are right about replacing `a11y` with accessibility here! I've added this change to the PR

**Pull request merged**